### PR TITLE
docs(readme): enhance images readmes

### PR DESCRIPTION
**Pull request by hleonardtek**
We should provide more relevant READMEs to our docker images.
Whoever is able to enhance any image's README should contribute to this PR.

### CODE REVIEW

**Comment by ad2ien** [comment]
@ccamel @SonnevilleDamien @amimart if you want the dockerhub page of the image you've been working on to be described, you can improve the corresponding readme file. (On this PR or not)

File: dockerfiles/cosmos-exporter/README.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
 # Cosmos-Exporter
 
 An exporter used to get blockchain metrics from the validators.
```
Comment (ad2ien) [inline dockerfiles/cosmos-exporter/README.md:3]:
```suggestion
An exporter used to get cosmos based blockchain metrics from a running node.
```

File: dockerfiles/cosmos-exporter/README.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+It queries nodes to extract informations concerning validators and wallets.
```
Comment (ad2ien) [inline dockerfiles/cosmos-exporter/README.md:4]:
```suggestion
It queries a node to extract informations concerning 
- The blockchain network in general
- Some specific validators and wallets.
```

**Review by ad2ien — CHANGES_REQUESTED** [review]
Nice readme, I added a suggestion to improve the intro.

**Comment by ccamel** [comment]
@hleonardtek great initiative. Same than @ad2ien did, I call also everyone involved (@SonnevilleDamien @amimart and me) to improve the `readme` files in this PR.

**Comment by ad2ien** [comment]
We add an image called cosmos-faucet made from [BitCannaCommunity/cosmos-faucet](https://github.com/BitCannaCommunity/cosmos-faucet) I removed it to be sure that okp4/cosmos-faucet is our faucet.

**Review by ad2ien — APPROVED** [review]
Let's go, It will then become a running task to make sure readme files are ok.

**Review by amimart — APPROVED** [review]

**Review by ccamel — APPROVED** [review]

**Pull request merged**